_sass/module/_candidate.scss

```diff
@@ -1,9 +1,23 @@
 $candidate-photo-width: 10rem;
 
-.candidate__occupation {
+.candidate__occupation__commitee_name {
   @include heading('h3');
   color: $subheading-color;
   font-style: italic;
+  font-size: 18px;
```

Comment:
We have should stick the [font scale](https://github.com/caciviclab/odca-jekyll/blob/master/_sass/base/_typography.scss#L1) (which may need some tweaking) instead of specifying absolute sizes here.